microovn/ovn/refresh.go

```diff
@@ -50,14 +44,6 @@ func refresh(s *state.State) error {
 		return fmt.Errorf("Failed to generate the daemon configuration: %w", err)
 	}
 
-	// Enable OVN central (if needed).
-	if hasCentral {
-		err = snapRestart("central")
```

Comment:
While this is OK for the NB and SB DB services, as they dynamically learn about their peers through the cluster, it is unfortunately not the case for `ovn-northd`.

```
root@micro1:~# ps axuw | grep ovn-northd
...
root        5377  0.0  0.0 164048  7168 ?        Ssl  15:49   0:00 ovn-northd --private-key=/var/snap/microovn/common/data/pki/ovn-northd-privkey.pem --certificate=/var/snap/microovn/common/data/pki/ovn-northd-cert.pem --ca-cert=/var/snap/microovn/common/data/pki/cacert.pem -vconsole:emer -vsyslog:err -vfile:info --ovnnb-db=ssl:10.215.82.73:6641 --ovnsb-db=ssl:10.215.82.73:6642 --no-chdir --log-file=/var/snap/microovn/common/logs/ovn-northd.log --pidfile=/var/snap/microovn/common/run/ovn/ovn-northd.pid --detach

root@micro2:~# ps axuw | grep ovn-northd
root        5461  0.0  0.0 164140  6144 ?        Ssl  15:49   0:00 ovn-northd --private-key=/var/snap/microovn/common/data/pki/ovn-northd-privkey.pem --certificate=/var/snap/microovn/common/data/pki/ovn-northd-cert.pem --ca-cert=/var/snap/microovn/common/data/pki/cacert.pem -vconsole:emer -vsyslog:err -vfile:info --ovnnb-db=ssl:10.215.82.73:6641,ssl:10.215.82.239:6641 --ovnsb-db=ssl:10.215.82.73:6642,ssl:10.215.82.239:6642 --no-chdir --log-file=/var/snap/microovn/common/logs/ovn-northd.log --pidfile=/var/snap/microovn/common/run/ovn/ovn-northd.pid --detach

root@micro3:~# ps axuw | grep ovn-northd
root        5459  0.0  0.0 164140  7168 ?        Ssl  15:49   0:00 ovn-northd --private-key=/var/snap/microovn/common/data/pki/ovn-northd-privkey.pem --certificate=/var/snap/microovn/common/data/pki/ovn-northd-cert.pem --ca-cert=/var/snap/microovn/common/data/pki/cacert.pem -vconsole:emer -vsyslog:err -vfile:info --ovnnb-db=ssl:10.215.82.73:6641,ssl:10.215.82.239:6641,ssl:10.215.82.41:6641 --ovnsb-db=ssl:10.215.82.73:6642,ssl:10.215.82.239:6642,ssl:10.215.82.41:6642 --no-chdir --log-file=/var/snap/microovn/common/logs/ovn-northd.log --pidfile=/var/snap/microovn/common/run/ovn/ovn-northd.pid --detach
```

Reply:
Oh, I thought we solved this already, but that was the `ovn-controller`, not the `northd`. Maybe we could separate the services so that one script doesn't handle all three services ([#2019832](https://bugs.launchpad.net/microovn/+bug/2019832))

Reply:
Yep, a split seems like a reasonable approach to me